### DISCUSSION TITLE
table filtering from button action form

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3362] [Q&A] [answered]
hey, we have following action in Segment:

<img width="754" alt="Zrzut ekranu 2023-07-31 o 09 09 07" src="https://github.com/twilio-labs/paste/assets/140410476/febdf6bc-b340-49e4-9615-c4bf8a4678cb">

Basically, we can control what to display in the table below with small form with checkboxes. Any ideas how to handle this case? Any alternatives suggestions? Thanks

**Comment by loreina** [comment, marked as answer]
Hi @marektonik,

We have a [Menu](https://paste.twilio.design/components/menu#multi-select-menu-items) component with multi-select items that would work well here. It's available in code, and in our backlog to add to Figma. We also have guidance on [composing menu items](https://paste.twilio.design/components/menu#composing-menu-items) to get closer to the screenshot you shared. Hope this helps!
